javalin/src/main/java/io/javalin/http/util/AsyncUtil.kt

```diff
@@ -18,6 +18,8 @@ internal object AsyncUtil {
 
     fun submitAsyncTask(context: Context, executor: ExecutorService?, timeout: Long, onTimeout: Runnable?, task: ThrowingRunnable<Exception>): Unit =
         context.future {
+            context.req().asyncContext.timeout = 0 // override default timeout system
```

Comment:
Might seem obvious to you, but I had to read your PR comment to understand why this is being done. Maybe mention `[timeout]` and `[onTimeout]` explicitly?